api/controller.py

```diff
@@ -187,6 +188,8 @@ def load_settings(self):
         new_top_level_lanes = {}
         # Create a CirculationAPI for each library.
         new_circulation_apis = {}
+        # Create a SharedCollectionAPI for each library.
+        new_shared_collection_apis = {}
```

Comment:
SharedCollectionAPI is meant to be similar to CirculationAPI. I'm not sure it's necessary, but I thought it might be useful if we ever want inter-library loan for other collection types.